src/olympia/constants/base.py

```diff
@@ -364,7 +364,7 @@
 
 # Changes to this list need to be synced with the privileged add-ons signing pipeline.
 # Please reach out to the Operations Team (awagner) before making any changes!
```

Comment:
add to this comment as it's not just for privileged add-ons now?

Reply:
Yeah, I wasn't sure exactly what to say here instead. Any suggestion @wagnerand ?

Reply:
I think we can say something like

```
# Changes to this list need to be coordinated with the Operations team to make sure other systems like the privileged add-ons signing pipeline are in sync. Please reach out to them before making changes.
```